crates/vm/levm/src/constants.rs

```diff
@@ -44,6 +45,12 @@ pub const MAX_BLOB_NUMBER_PER_BLOCK: usize = 6;
 pub const TARGET_BLOB_GAS_PER_BLOCK: U256 = U256([393216, 0, 0, 0]); // TARGET_BLOB_NUMBER_PER_BLOCK * GAS_PER_BLOB
 pub const MIN_BASE_FEE_PER_BLOB_GAS: U256 = U256([1, 0, 0, 0]);
 pub const BLOB_BASE_FEE_UPDATE_FRACTION: U256 = U256([3338477, 0, 0, 0]);
+pub const MAX_BLOB_COUNT: usize = 6;
+pub const VALID_BLOB_PREFIXES: [u8; 2] = [0x01, 0x02];
+// Storage constants
+pub const COLD_STORAGE_ACCESS_COST: U256 = U256([2100, 0, 0, 0]);
+pub const WARM_ADDRESS_ACCESS_COST: U256 = U256([100, 0, 0, 0]);
+pub const BALANCE_COLD_ADDRESS_ACCESS_COST: U256 = U256([2600, 0, 0, 0]);
```

Comment:
This cost constants should be in `gas_cost.rs`, since they are mainly used in that file